src/StudioCore/Assets/Paramdex/ER/Upgrader/massedit.txt

```diff
@@ -48,4 +48,529 @@ param ToughnessParam: added: unk1: = 1
 param ToughnessParam: modified && !added: unk1: = vanillafield unk1
 11001000L
 1.10 - (ToughnessParam) Set unk2 to Vanilla v1.10 values
-param ToughnessParam: modified && !added: unk2: = vanillafield unk2
\ No newline at end of file
+param ToughnessParam: modified && !added: unk2: = vanillafield unk2
```

Comment:
param SwordArtsParam: modified: swordArtsTypeNew: = field swordArtsType will stop functioning if the name is changed again (should re-check if other unks were identified too)